app/models/requests/DataRequest.scala

```diff
@@ -50,4 +52,19 @@ case class DataRequest[A](request: UserRequest[A],
       logger.debug(s"[dispatchPlace] Invalid value for DISPATCH_PLACE: $value")
       None
   }
+
+  def isUkToUkAndNoGuarantor: Boolean =
+    (userAnswers.get(DestinationTypePage), userAnswers.get(GuarantorRequiredPage)) match {
+      case (Some(scenario), Some(false)) if scenario.movementType(this) == MovementType.UkToUk => true
+      case _ => false
+    }
+
+  def isUkToEuAndNoGuarantor: Boolean =
+    (userAnswers.get(DestinationTypePage), userAnswers.get(GuarantorRequiredPage)) match {
+      case (Some(scenario), Some(false)) if scenario.movementType(this) == MovementType.UkToEu => true
+      case _ => false
+    }
```

Comment:
I'm not sure this check is right, shouldn't this be only for NI to EU? Also, there are other movement types which would count as NI to EU which aren't covered by the `MovementType.UKtoEU`

Reply:
Yeah I copied the existing code, I'll double check if it's actually right though with Lexi/Haresh.